hardhat-example/README.md

```diff
@@ -0,0 +1,25 @@
+# Hardhat example
+
+Simple scripts for basic operations like hbars transfer, balance fetching, and contract interactions (deployment and calls).
+
+## Prerequisite
+You must have running:
+- JSON-RPC Relay on port :7546
+- Hedera Local Node
```

Comment:
Does it have to be `Local Node`?
Fine for this PR but if there's a reason you could point it to a remote env please call that out so we can get a ticket and address it later